packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.tsx

```diff
@@ -48,10 +48,11 @@ export const DetailsRowCheck = (props: IDetailsRowCheckProps) => {
       aria-checked={ isPressed }
       data-selection-toggle={ true }
       data-automationid='DetailsRowCheck'
+      type='button'
```

Comment:
Declaring a `<button>` as `type='button'` is redundant.

Reply:
Not exactly: https://www.w3.org/TR/2012/WD-html-markup-20121011/button.button.html

Reply:
It defaults to `type="submit"` -- https://www.w3.org/TR/html5/sec-forms.html#element-attrdef-button-type

But typically this only matters in the context of a form if you're trying to negate the submit behavior.

Reply:
Interesting... https://twitter.com/davidmacd/status/989170029961859073

Reply:
So do we keep it or toss it? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button confirms that it defaults to 'submit'